src/screens/ConnectSignUpScreen.js

```diff
@@ -0,0 +1,169 @@
+import React, { Component } from 'react';
+import { Font } from 'expo';
+import { imageStyles } from '../styles/images';
+import { layoutStyles, margins } from '../styles/layout';
+import Icon from '@expo/vector-icons/FontAwesome';
+import { AppRegistry, Text, StyleSheet, TextInput, View, ScrollView, TouchableHighlight } from 'react-native';
+import { Form, t } from '../components/Form';
+import Button from '../components/Button';
+import ConnectSignUpRequester from '../helpers/requesters/ConnectSignUpRequester';
+
+export default class ConnectSignUp extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      formValues: this.getInitialFormValues(),
+      errors: {},
+    };
+
+    this.onFormChange = this.onFormChange.bind(this);
+    this.signUp = this.signUp.bind(this);
+    this.navigateToConnect = this.navigateToConnect.bind(this);
+  }
+
+  /**
+    * Returns params for default values of the form
+    */
+  getInitialFormValues() {
+    return {
+      acceptMessages: true,
+      shareProfile: true,
+      acceptNotices: true,
+    };
+  }
+
+  /**
+    * Defines the field types of the form
+    */
+  getFormType() {
+    return t.struct({
+      militaryBranch: t.enums({
+        1: 'Army',
+        2: 'Navy',
+        3: 'Marines',
+        4: 'Air Force',
+        5: 'Coast Guard',
+        6: 'First Responder',
```

Comment:
we should be getting the roles and militaryBranch from the enums defined in the veteran model